client/tests/integration/smartcontracts/create_nft_for_every_user_trigger/src/lib.rs

```diff
@@ -10,11 +8,11 @@ extern crate panic_halt;
 
 use alloc::{format, string::ToString};
 
-use iroha_wasm::{data_model::prelude::*, prelude::*};
+use iroha_trigger::prelude::*;
 
-#[iroha_wasm::main]
-fn main() {
-    iroha_wasm::info!("Executing trigger");
+#[iroha_trigger::main]
+fn main(_owner: AccountId, _event: Event) {
```

Comment:
Why arguments are unpacked from payload?

Reply:
Ok, it's current approach to unpack payloads, which might be changed in the future.

Reply:
After a discussion with Roman in DM decided to move it into #3819